app/components/HeaderBlock.vue

```diff
@@ -55,27 +55,27 @@ export default {
 
       // refresh arcsiShows every 10 minutes
       if (minutes % 10 === 0) {
-        this.refreshArcsiShows()
+        this.refreshAllShows()
       }
 
-      // refresh arcsiShows every 3 minutes
+      // refresh rareShows every 3 minutes
       if (minutes % 3 === 0) {
         this.refreshRareShows()
       }
 
-      // refresh arcsiShows every 4 minutes
+      // refresh customSchedule every 4 minutes
       if (minutes % 4 === 0) {
         this.refreshCustomSchedule()
       }
     },
-    async refreshArcsiShows () {
-      await this.$axios.get(arcsiServerURL)
+    async refreshAllShows () {
+      await this.$axios.get(arcsiShowsBaseURL + '/all_without_items')
         .then((res) => {
-          this.$store.commit('refreshArcsiShows', res.data)
+          this.$store.commit('refreshAllShowsList', res.data)
         })
         .catch((e) => {
           this.$sentry.captureException(e)
-          this.error({ statusCode: 404, message: 'Latest news not found' })
+          this.error({ statusCode: 404, message: 'All shows endpoint not found' })
```

Comment:
The exception message on line 98 should be updated too.

Reply:
Good catch! Fixed in 53af75fd4556aa883f6e6e383875f4f22218be0b